config/samples/deployment.yaml

```diff
@@ -126,5 +126,13 @@ spec:
           requests:
             cpu: 500m
             memory: 256Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          privileged: false
+          readOnlyRootFilesystem: true
+      securityContext:
+        fsGroup: 4059
+        runAsGroup: 4059
+        runAsUser: 4059
```

Comment:
The `emptyDir` is missing here. Could you add it, I assume that otherwise the operator won't work since `readOnlyRootFilesystem` is set.

Reply:
Done by running `make clean all` :)